### PR TITLE
Note that `az login` works for Ansible, too.

### PR DESCRIPTION
**Pull request by mavit**
##### SUMMARY
Explain that one can use the `az login` command from the Azure CLI to authenticate Ansible to Azure.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
Azure modules.

### CODE REVIEW

**Comment by acozine** [comment]
@mavit thanks for contributing to the Ansible documentation! 

Do you use an Ansible task to sign in via the Azure CLI? Are you managing your Azure authentication outside of Ansible?

**Comment by mavit** [comment]
For now I'm running `az login` before `ansible-playbook`.  This has the advantage that I don't need to fiddle around with storing shared secrets in Ansible Vault.

File: lib/ansible/plugins/doc_fragments/azure.py [hunk 1]
```diff
@@ -93,14 +93,19 @@ class ModuleDocFragment(object):
     - azure >= 2.0.0
 
 notes:
-    - For authentication with Azure you can pass parameters, set environment variables or use a profile stored
-      in ~/.azure/credentials. Authentication is possible using a service principal or Active Directory user.
-      To authenticate via service principal, pass subscription_id, client_id, secret and tenant or set environment
+    - For authentication with Azure you can pass parameters, set environment variables, use a profile stored
+      in ~/.azure/credentials, or log in with C(az login). Authentication is possible using a service principal or Active Directory user.
```
Comment (acozine) [inline lib/ansible/plugins/doc_fragments/azure.py]:
```suggestion
      in ~/.azure/credentials, or log in before you run your tasks or playbook with C(az login). Authentication is possible using a service principal or Active Directory user.
```

**Comment by acozine** [comment]
Thanks for the clarification @mavit - if you'll accept my suggestion, I'll merge this PR.

**Comment by acozine** [comment]
@mavit I'll fix the test failure and merge, since it was my suggestion that caused the failure in the first place

**Pull request merged**